src/body.js

```diff
@@ -39,9 +39,11 @@ export default function Body(body, {
 	} else if (body instanceof Blob) {
 		// body is blob
 	} else if (Buffer.isBuffer(body)) {
-		// body is buffer
-	} else if (Object.prototype.toString.call(body) === '[object ArrayBuffer]') {
-		// body is array buffer
+		// body is ArrayBuffer
+	} else if (body instanceof ArrayBuffer) {
```

Comment:
Was there a reason to change the ArrayBuffer check semantics? This makes it impossible to use e.g. ArrayBuffer from a VM context, while ArrayBufferView from a VM context would work fine.

Reply:
I'm not sure what you mean by VM context here. `instanceof` looks cleaner and is consistent with the rest of checks like `body instanceof Blob` or `body instanceof Stream`, this was the primary reason for the change.

Reply:
```js
const vm = require('vm');
const ForeignArrayBuffer = vm.runInNewContext('ArrayBuffer');
const req = new Request('about:blank', {
  method: 'POST',
  body: new ForeignArrayBuffer(1)
});
```

`ArrayBuffer.isView` works on objects from VM contexts, so it would seem odd for `ArrayBuffer` objects to not support that.

In browsers, this behavior is observable through `iframe` contexts.

Reply:
Thanks for explanation, I think a test case for this tricky scenario should be added.